code/modules/projectiles/guns/projectile_gun.dm

```diff
@@ -79,14 +79,21 @@
 	return !magazine
 
 /obj/item/gun/projectile/proc/reload(obj/item/ammo_box/magazine/AM, mob/user as mob)
-		user.remove_from_mob(AM)
-		magazine = AM
-		magazine.loc = src
-		playsound(src, magin_sound, 50, 1)
-		chamber_round()
-		AM.update_icon()
-		update_icon()
+	user.remove_from_mob(AM)
+	magazine = AM
+	magazine.loc = src
```

Comment:
would you make this use forcemove instead

Reply:
I think I fixed it